repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/deploy/ContainerConfigurationListener.java

```diff
@@ -56,14 +56,11 @@ public synchronized void configurationUpdated(ContainerConfiguration configurati
                 if (ad.getCheckInterval() > 0) {
                     dirWatcher.updateCheckInterval(ad.getCheckInterval());
                 }
-
-                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue()));
-
-                deploymentDirectory = new File(dd.getValue());
+                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue().trim()));
+                deploymentDirectory = new File(dd.getValue().trim());
```

Comment:
It looks like these directories do not, technically, have to be absolute. Since spaces (which `trim()` removes) are valid directory name characters (on Mac OS X, at least), I'm worried that these lines limit support on certain file systems.

While I would be happy to assist the user where possible, in this case doing so limits our versatility. Ultimately, I want to say that using whitespace to format XML text nodes is not a good idea, and it's on the user to make sure they are providing the value they want.

@adrianjgeorge Thoughts?